tests/.infra/centos-ci/functional_tests_utils.sh

```diff
@@ -405,4 +405,22 @@ function createIndentityProvider() {
   CHE_OPENSHIFT_PROJECT=eclipse-che
   keycloakPodName=$(oc get pod --namespace=$CHE_OPENSHIFT_PROJECT | grep keycloak | awk '{print $1}')
   /tmp/oc exec $keycloakPodName --namespace=$CHE_OPENSHIFT_PROJECT -- /opt/jboss/keycloak/bin/kcadm.sh create identity-provider/instances -r che -s alias=github -s providerId=github -s enabled=true -s storeToken=true -s addReadTokenRoleOnCreate=true -s 'config.useJwksUrl="true"' -s config.clientId=$CHE_MULTI_USER_GITHUB_CLIENTID_OCP -s config.clientSecret=$CHE_MULTI_USER_GITHUB_SECRET_OCP -s 'config.defaultScope="repo,user,write:public_key"' --no-config --server http://localhost:8080/auth --user admin --password admin --realm master
-}
\ No newline at end of file
+}
+
+function runDevfileTestSuite() {
+  defindCheRoute
+  ### Create directory for report
+  mkdir report
+  REPORT_FOLDER=$(pwd)/report
+  ### Run tests
+  docker run --shm-size=1g --net=host  --ipc=host -v $REPORT_FOLDER:/tmp/e2e/report:Z \
```

Comment:
Will the job fail, if the tests fail? Can you confirm this?

Reply:
No the  job is not fails we should improve it for happy path and devfiles jobs.

Reply:
@musienko-maxim It's up to you if you implement this as part of this PR, or you'll create an issue for implementation of this. Your call.

Reply:
i've created issue: https://github.com/eclipse/che/issues/16030

Reply:
Thx!